core/src/main/java/hudson/FilePath.java

```diff
@@ -2280,8 +2284,7 @@ private void readFromTar(String name, File baseDir, InputStream in) throws IOExc
                     if (parent != null) mkdirs(parent);
                     writing(f);
 
-                    byte linkFlag = (Byte) LINKFLAG_FIELD.get(te);
-                    if (linkFlag==TarEntry.LF_SYMLINK) {
+                    if (te.isSymbolicLink()) {
```

Comment:
No, that deletes `LINKNAME_FIELD`. https://github.com/oleg-nenashev/jenkins/blob/JENKINS-10629/core/src/main/java/hudson/FilePath.java#L2726-2738